### DISCUSSION TITLE
Does Renovate generate release notes when exclusively using LockFileMaintenance?

### DISCUSSION THREAD

**Discussion by Duroth** [renovatebot/renovate#12327] [Request Help] [answered]
Renovate combines release notes for all updated (and skipped) versions -- but does this feature also work when exclusively performing Lock File Maintenance (i.e. not updating the package files)?

**Comment by HonkingGoose** [comment, marked as answer]
Renovate does not show changelogs for lockfile updates.

I tried designing a lockfile diff view for Renovate, and got horribly stuck. So this is not an easy thing to add, at least not for me... 🙈 [^stuck]
We have a issue tracking "lock file diffs". [^lock-diff] 

Did you know that GitHub can show you a diff of the changes in a lockfile PR for some programming languages? [^gh-docs]

[^lock-diff]: https://github.com/renovatebot/renovate/issues/651
[^stuck]: https://github.com/renovatebot/renovate/issues/7536
[^gh-docs]: https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-dependency-changes-in-a-pull-request
